apps/docs/content/guides/platform/manage-your-usage/disk-size.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: 'manage-usage-disk-size'
+title: 'Manage Disk size usage'
+---
+
+## What you are charged for
+Each Supabase project has a dedicated [disk](/docs/guides/platform/compute-and-disk#disk). You are charged for the provisioned disk size.
```

Comment:
Technically it's per databases - when you run multiple databases (primary+RRs), each database has it's own disk